app/javascript/components/pages/AboutUs.js

```diff
@@ -33,9 +42,9 @@ class AboutUs extends Component {
 												<br></br>
 												<div>
 													<ListGroupItemText> <h1 style={{ textAlign: 'center', fontSize: '3rem' }}>Sami</h1></ListGroupItemText>
-                                                    <h3> <a href= "https://www.linkedin.com/in/mabrouk10/"> < FaLinkedin /> </a> 
+                                                    <h3> <a id= 'icons' href= "https://www.linkedin.com/in/mabrouk10/"> < FaLinkedin /> </a> 
```

Comment:
ids should be used uniquely on a page, but it looks like it's being used twice. Perhaps, change icon from id to class to apply styles to both anchor tags